vcpkg/users/versioning.md

```diff
@@ -5,8 +5,7 @@ ms.date: 11/30/2022
 ---
 # Versioning reference
 
-Versioning allows you to deterministically control the precise revisions of dependencies used by
-your project from within your manifest file. Versioning only applies to [Manifest mode](manifests.md).
+Versioning allows you to deterministically control the precise revisions of dependencies used by your project from within your manifest file. Versioning only applies to [Manifest mode](manifests.md).

```

Comment:
Suggest rewording last sentence to: "Versioning is only available to Manifest mode users".